### PR TITLE
EntityAssignTest: testReorderRecords

### PR DESCRIPTION
**Pull request by MichaelMicrof**
https://trello.com/c/7nsHqhsE/264-at06701java-entity-assign-reorder-records

### CODE REVIEW

File: src/test/java/EntityAssignTest.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        Assert.assertEquals(getIcon(EntityAssignConstants.ASSIGN_GET_ICON)
+                .getAttribute("class"), EntityAssignConstants.CLASS_ITEM_SAVE_DRAFT);
+
+        toggleAction(EntityAssignConstants.ASSIGN_TOGGLE_LIST_ACTION);
+
+        Assert.assertEquals(getIcon(EntityAssignConstants.ASSIGN_GET_ICON)
+                .getAttribute("class"), EntityAssignConstants.CLASS_ITEM_SAVE_DRAFT);
+    }
+}
+
```
Comment (SergeiKuznetsov) [inline src/test/java/EntityAssignTest.java:74]:
```suggestion
```

File: src/test/java/constants/EntityAssignConstants.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    public static final By ASSIGN_BUTTON_SAVE_DRAFT = By.id("pa-entity-form-draft-btn");
+
+    public static final By ASSIGN_TOGGLE_LIST_ACTION = By.xpath("//a[@href=\"index.php?action=action_list&list_type=table&entity_id=37\"]");
+    public static final By ASSIGN_TOGGLE_ORDER_ACTION = By.xpath("//a[@href=\"index.php?action=action_list&list_type=table&entity_id=37&draggable=1\"]");
+
+    public static final By ASSIGN_ROW_SAVE_RECORD = By.id("customId_0");
+    public static final By ASSIGN_GET_ICON = By.xpath("//tbody/tr[1]/td[1]/i");
+
+    public static final String CLASS_ITEM_SAVE_DRAFT = "fa fa-pencil";
+
```
Comment (SergeiKuznetsov) [inline src/test/java/constants/EntityAssignConstants.java:23]:
```suggestion
```

**Review by SergeiKuznetsov — CHANGES_REQUESTED** [review]

File: src/test/java/EntityAssignTest.java (hunk 1 above)
Comment (MichaelMicrof) [reply src/test/java/EntityAssignTest.java]: Done

File: src/test/java/constants/EntityAssignConstants.java (hunk 2 above)
Comment (MichaelMicrof) [reply src/test/java/constants/EntityAssignConstants.java]: done

**Review by SergeiKuznetsov — APPROVED** [review]

**Review by ValeriiSarukhanov — APPROVED** [review]

**Review by RobinHooD1024 — COMMENTED** [review]
Looks good for me

**Pull request merged**